python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -480,6 +483,91 @@ def close(self) -> None:
         self._worker.core_worker.experimental_channel_set_error(self._reader_ref)
 
 
+@DeveloperAPI
+class BufferedSharedMemoryChannel(ChannelInterface):
```

Comment:
We need to support serialization (`__reduce__`) , right? Can you add a test for it?

Reply:
technically it should just work. but nothing to lose by adding a simple test

Reply:
tested